src/Commands/MakeQueryBuilderCommand.php

```diff
@@ -8,14 +8,15 @@
 
 class MakeQueryBuilderCommand extends Command
 {
-    protected $signature = 'beyond:make:query-builder {name}';
+    protected $signature = 'beyond:make:query-builder {name} {--overwrite}';
```

Comment:
You did eveywhere `{--overwrite=false}` but here it is missing. Is `=false` even required?